README.md

```diff
@@ -4,46 +4,81 @@
+    `s3fs`,  `aws-java-sdk-s3`, `tika-core`) under the `lib/server` directory
+    of OMERO.server and restarting the server,
+
+-   or by building a modified version of the OMERO.server binary after declaring
+    `com.glencoesoftware.omero:omero-zarr-pixel-buffer` as a dependency of the
+    [Ivy build system](https://github.com/ome/openmicroscopy/blob/develop/ivy.xml).
+
+## Usage
 
-After putting the `omero-zarr-pixel-buffer.jar` under `lib/server`, the server
-must be restarted. To use the ZarrPixelBuffer, an Image or Mask object must be populated with
-an [ExternalInfo](https://docs.openmicroscopy.org/omero-blitz/latest/slice2html/omero/model/ExternalInfo.html)
-object with the following properties:
+The pixel buffer can read Zarr data generated by the [bioformats2raw](https://github.com/glencoesoftware/bioformats2raw) utility
+and meeting the following requirements:
+
+-   the Zarr dataset must be conformant with the
+    [Zarr v2 specification](https://zarr-specs.readthedocs.io/en/latest/v2/v2.0.html)
+    and the [version 0.4 of the NGFF specification](https://ngff.openmicroscopy.org/0.4/)
+-   the image (or label image) must have between 2 and 5 declared dimensions
+    of type space, channel or time. Supported dimensions are: (y, x), (t, y, x),
+    (c, y, x), (z, y, x), (t, c, y, x), (t, z, y, x), (c, z, y, x) and
+    (t, c, z, y, x)
+-   the resolution levels must be downsampled alongside the spatial dimensions,
+    either alongside the (y, x) or the (z, y, x) axes
+-   the scaling factor alongside both the y and the x axes between successive resolution
+    levels must be equal to two
+
+An OMERO Image or Mask object can be associated to Zarr by setting its
+[ExternalInfo](https://docs.openmicroscopy.org/omero-blitz/5.8.3/slice2html/omero/model/ExternalInfo.html)
+with the following properties:
 
 -   `entityType` must be set to `com.glencoesoftware.ngff:multiscales`
 -   `entityId` must be set to `3`
--   `lsid` must be set to the location of the corresponding Zarr multiscales group:
-    - either the absolute path to the multiscales group if the Zarr is on the
-      filesystem e.g. `/data/CMU-1.ome.zarr/0`
-    - or the URI to the multiscales group if the Zarr is on Amazon S3 defined
-      as `s3://s3.<region>.amazonaws.com/<bucket>/<key>` e.g.
+-   `lsid` must be set to the path of the Zarr group describing the
+     [multiscales](https://ngff.openmicroscopy.org/0.4/#multiscale-md) image
+     or label image:
+
+    - either the absolute path to the multiscales group if the Zarr is stored on
+      the filesystem e.g. `/data/CMU-1.ome.zarr/0`
+    - or the URI specifying the location of the multiscales group if the Zarr is
+      stored on AWS S3 or S3 compatible storage. The location must be specified
+      as `s3://<endpoint>/<bucket>/<prefix>` where `<endpoint>` is the S3
+      endpoint - see https://docs.aws.amazon.com/general/latest/gr/s3.html#s3_region
```

Comment:
Two notes here:

1. If the expectation is that users will be manually setting the `lsid`, I think we should mention that at least for S3 URIs we need them to be URL encoded.
2. We might want to make it clear that the URI will probably end with the series as we do in the local filesystem example. People might expect the URI for their Zarr image would end in `.zarr`.